### PR TITLE
netpbm: fix RGB path config

### LINKED ISSUE #99802: netpbm needs to install an additional file
### `brew gist-logs <formula>` link OR `brew config` AND `brew doctor` output

```shell
758> brew gist-logs netpbm
Error: No logs.
```

```
759> brew config
HOMEBREW_VERSION: 3.4.7
ORIGIN: https://github.com/Homebrew/brew
HEAD: 09e766f83c8a420cce9e2210c46a24ec3864a094
Last commit: 4 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: a3697d744c9a81075f24ead86dcbceb75acb7fd3
Core tap last commit: 38 minutes ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_CORE_GIT_REMOTE: https://github.com/Homebrew/homebrew-core
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.IixFQOl4vt/org.xquartz:0
HOMEBREW_EDITOR: vi
HOMEBREW_MAKE_JOBS: 16
Homebrew Ruby: 2.6.8 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 16-core 64-bit kabylake
Clang: 13.1.6 build 1316
Git: 2.31.1 => /usr/local/bin/git
Curl: 7.77.0 => /usr/bin/curl
macOS: 12.1-x86_64
CLT: 13.3.0.0.1.1645755326
Xcode: 13.3 => /Applications/Xcode-v13.3.app/Contents/Developer
```
```


### Verification

- [X] I ran `brew update` and am still able to reproduce my issue.
- [X] I have resolved all warnings from `brew doctor` and that did not fix my problem.

### What were you trying to do (and why)?

Trying to use `xpmtoppm` (part of netpbm) on an xpm file that uses color names:
` ``
xpmtoppm ./favicon.xpm  > favicon.ppm
```

Here is the small input xpm file `favicon.xpm`:
```
/* XPM */
static char * favicon_xpm[] = {
"16 16 4 1",
" 	c white",
".	c blue",
"X	c black",
"o	c red",
"                ",
"                ",
"                ",
"                ",
"  ....    ....  ",
" .    .  .    . ",
".  ..  ..  ..  .",
"  .  . .. .  .  ",
" .   XXXXXX   . ",
" .   XXXXXX   . ",
"oooooooooooooooo",
"oooooooooooooooo",
"oooooooooooooooo",
"oooooooooooooooo",
"XXXXXXXXXXXXXXXX",
"XXXXXXXXXXXXXXXX"};
```

### What happened (include all command output)?

```
 764> xpmtoppm ./favicon.xpm > favicon.ppm
xpmtoppm: can't open color names dictionary file from the path '/usr/local/netpbm/rgb.txt:/usr/share/netpbm/rgb.txt:/etc/X11/rgb.txt:/usr/lib/X11/rgb.txt:/usr/share/X11/rgb.txt:/usr/X11R6/lib/X11/rgb.txt' and Environment variable RGBDEF not set.  Set RGBDEF to the pathname of your rgb.txt file or don't use color names.
```

### What did you expect to happen?

I expected the conversion from xpm to ppm to work (i.e. create `favicon.ppm` as a valid ppm file) and not produce the above error message. 

The root cause appears to be that I'm missing a link from `/usr/local/netpbm/rgb.txt` to `/usr/local/Cellar/netpbm/10.86.32/misc/rgb.txt`.  This is surely only about a 1-2 line adjustment to the formula but I'm not up on ruby and how to make this change.  I would expect it to be in this section:  https://github.com/Homebrew/homebrew-core/blob/728d76ba9e69f33a0f0d5f2fc0550529f4247bfa/Formula/netpbm.rb#L71-L73

### Step-by-step reproduction instructions (by running `brew` commands)

```shell
Save the XPM in the above answer to `favicon.xpm` noting that it uses names for colors instead of rgb values and then run:

brew install netpbm
xpmtoppm ./favicon.xpm > favicon.ppm
```
note that the error message indicates `/usr/local/netpbm/rbg.txt` is the first place checked and note that there is a `rgb.txt` in the cellar for this package.
```

### PR DESCRIPTION
**Pull request by gromgit**
Also added test to verify setting. Fixes #99802.

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

### CODE REVIEW

File: Formula/netpbm.rb [hunk 1]
```diff
@@ -46,6 +47,7 @@ def install
       s.change_make_var! "ZLIB", "-lz"
       s.change_make_var! "JASPERLIB", "-ljasper"
       s.change_make_var! "JASPERHDR_DIR", "#{Formula["jasper"].opt_include}/jasper"
+      s.gsub! "/usr/local/netpbm/rgb.txt", "#{opt_prefix}/misc/rgb.txt"
```
Comment (carlocab) [inline Formula/netpbm.rb:50]: Why `opt_prefix` and not `prefix`?

File: Formula/netpbm.rb [hunk 2]
```diff
@@ -79,5 +81,32 @@ def install
...
+      " .   XXXXXX   . ",
+      " .   XXXXXX   . ",
+      "oooooooooooooooo",
+      "oooooooooooooooo",
+      "oooooooooooooooo",
+      "oooooooooooooooo",
+      "XXXXXXXXXXXXXXXX",
+      "XXXXXXXXXXXXXXXX"};
+    EOS
+    system "#{bin}/xpmtoppm test.xpm > test.ppm"
```
Comment (carlocab) [inline Formula/netpbm.rb:109]: `shell_output` seems more appropriate here than redirecting stdout.

File: Formula/netpbm.rb (hunk 1 above)
Comment (gromgit) [reply Formula/netpbm.rb]: Force of habit, sorry.

Comment (danmcmahill) [reply Formula/netpbm.rb]: Thanks for updating the path.  I like that better than adding an extra link as I had originally suggested.

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**